docs/sources/next/javascript-api/k6-browser/elementhandle/setchecked.md

```diff
@@ -0,0 +1,69 @@
+---
+title: 'setChecked(checked[, options])'
+description: 'Browser module: elementHandle.setChecked method'
+---
+
+# setChecked(checked[, options])
+
+{{< admonition type="warning" >}}
+
+Use [`locator.setChecked(checked [, options])`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/locator/setchecked/) instead.
+
+{{< /admonition >}}
+
+Use this method to select or unselect an `input` or `radio` checkbox.
+
+<TableWithNestedRows>
```

Comment:
```suggestion
```

I don't think we need this HTML element here.

Reply:
We use this everywhere in the browser docs because we have "options" and "options.force" (nested). Do you think we should remove all `TableWithNestedRows` from our documentation? Do you know when we should use it? Thanks!

Reply:
Ah, I think I see what you mean. In the old docs this would actually render the table differently:

<img width="840" alt="Screenshot 2024-08-29 at 11 40 39 AM" src="https://github.com/user-attachments/assets/c583a8d1-45d3-4aa1-a2bc-4db8e7d4eaf9">

But in the Grafana docs, I guess we don't have that element and so it doesn't really make a difference in how the table is rendered:

<img width="812" alt="Screenshot 2024-08-29 at 11 41 33 AM" src="https://github.com/user-attachments/assets/2f51a1a8-367b-45fc-b72e-9d5c21f67547">

So I guess we'd have to decide if this is something that we'd like to see in the Grafana docs, and then ask for help from the website team, or if we're ok with how it's rendered today.

Reply:
Thanks for the examples. I've created an issue with your reply 😄 https://github.com/grafana/website/issues/21482. So, I'm going to merge this PR. Once we decide on `TableNestedView`, we can remove the tag in one go or maybe leave it as it is.